cms/static/cms/js/modules/cms.page_type_select.js

```diff
@@ -0,0 +1,26 @@
+//##############################################################################
```

Comment:
It would make sense to centralize all wizard related js in one file. Once it gets bigger > than separate